Amplify/Categories/Predictions/Request/Identify/Identify.swift

```diff
@@ -0,0 +1,50 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+/// Identification criteria provided to
+/// type parameter in identify() API
+extension Predictions {
+    public enum Identify {
+        public struct Request<Output> {
+            @_spi(PredictionsIdentifyRequestKind)
+            public let kind: Kind
+        }
+
+        public struct Options {
+            /// The default NetworkPolicy for the operation. The default value will be `auto`.
+            public let defaultNetworkPolicy: DefaultNetworkPolicy
+            /// Extra plugin specific options, only used in special circumstances when the existing options do not provide
+            /// a way to utilize the underlying storage system's functionality. See plugin documentation for expected
+            /// key/values
+            let pluginOptions: Any?
```

Comment:
Should this be `public` as well, for consistency with `Predictions.Interpret.Options`?

Reply:
Addressed in 2d5a4650a7e0e76049a0fc83c545e700b0a55b5d